bindings/gumjs/gumdukcore.c

```diff
@@ -2680,7 +2683,7 @@ gumjs_native_function_init (duk_context * ctx,
 
     if (is_marker)
     {
-      if (is_variadic)
+      if (i == 0 || is_variadic)
```

Comment:
Hmm, isn't this allowed by the ABI? The implementation's chosen convention could be that the last value is a magic value.

Reply:
According to source of libffi, variadic function must have at least fixed argument. Below is the source of function `ffi_prep_cif_core`:

```

  FFI_ASSERT(cif != NULL);
  FFI_ASSERT((!isvariadic) || (nfixedargs >= 1));
  FFI_ASSERT(nfixedargs <= ntotalargs);
``` 

So i think maybe it is a bug of frida-gum.

Reply:
Ahh, nice catch!